pipelines/katello_pipeline.yml

```diff
@@ -19,7 +19,7 @@
     - role: disable_firewall
     - role: disable_ipv6
     - role: foreman_server_repositories
-      foreman_server_repositories_katello_client: katello_version == 'nightly' or katello_version is number and katello_version is version('3.9', '>=') | bool
+      foreman_server_repositories_foreman_client: katello_version == "nightly" or katello_version is version('3.9', '>=') | bool
```

Comment:
Should we have a tri-state value for this parameter? true (always on), auto (on if the version is new enough) or false (always off). Then we don't need to duplicate all when code.

Reply:
I'll think about this

Reply:
@ekohl how about this? instead of making weird states, I just made `foreman_client_repositories` (which is what gets loaded when you pass `foreman_server_repositories_foreman_client: true`) to be a NOOP on pre-1.20.